catalog/rest/catalog-rest-impl/src/main/java/org/codice/ddf/rest/impl/CatalogServiceImpl.java

```diff
@@ -548,6 +548,14 @@ private BinaryContent createMetacard(
 
     } catch (MetacardCreationException | CatalogTransformerException e) {
       throw new CatalogServiceException("Unable to create metacard");
+    } finally {
+      try {
+        if (stream != null) {
+          stream.close();
+        }
+      } catch (IOException e) {
+        LOGGER.error("Unexpected error closing stream", e);
```

Comment:
✏️ It may be a good idea to log this at a lesser level. I think I remember being told that not being able to close a stream isn't that big of a deal.

Can anyone confirm this?

Reply:
Could be changed to a warning. Note that https://github.com/codice/ddf/pull/3536 is already merged though